Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'puma', '~> 3.0'
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 gem 'active_model_serializers', '~> 0.10.0'
+gem 'simplecov', :require => false, :group => :test
 group :development, :test do
```

Comment:
it's in the test environment because it's group points to test. it would probably cleaner if moved there, though